tutorials/config/settings.py

```diff
@@ -33,7 +33,7 @@
                     'descr': 'Login nodes',
                     'scheduler': 'local',
                     'launcher': 'local',
-                    'environs': ['gnu', 'intel', 'pgi', 'cray'],
+                    'environs': ['gnu', 'intel', 'pgi', 'cray', 'builtin'],
```

Comment:
By adding `builtin` here, we will need to update the explanations in the current tutorial. Do you want to do that, too?

Reply:
Yes, I can change it. Let me have a look

Reply:
@vkarak Actually, looking at the changes that we have to make in the tutorial maybe it makes more sense to download the benchmarks in one of the `['gnu', 'intel', 'pgi', 'cray']`, what do you think?

Reply:
Haha, I knew that :-) Yes, just pick one!

Reply:
Okay, fixed it 😄 and I also fixed the `.. versionadded::` in the documentation